_docs/tasks/installing-istio.md

```diff
@@ -37,14 +37,20 @@ default namespace. They can be modified for deployment in a different namespace.
 
 1. Go to the [Istio release](https://github.com/istio/istio/releases) page, to download the installation file corresponding to your OS.
 
-2. Extract the installation file, and change directory to the location where the files were extracted. Following instructions are relative to this installation directory.
+1. Extract the installation file, and change directory to the location where the files were extracted. Following instructions are relative to this installation directory.
    The installation directory contains:
     * yaml installation files for Kubernetes
     * sample apps
     * the `istioctl` client binary, needed to inject Envoy as a sidecar proxy, and useful for creating routing rules and policies.
-    * a text file containing the Istio version.
+    * the istio.VERSION configuration file.
 
-3. Run the following command to determine if your cluster has [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/)
+1. Add the `istioctl` client to your PATH. For example, run the following commands on a Mac system:
+
+   ```bash
+   sudo ln -s $PWD/istioctl /usr/local/bin/
```

Comment:
this is kinda ugly. maybe we can just add the directory to the path instead
```shell
export PATH=$PATH:`pwd`
```
that would work if we put more tools later too

this being said maybe we should but the isoctl binary in a `bin/` sub directory